### PR TITLE
childchain spec

### LINKED ISSUE #4: design doc for child chain
# Bounty

Specify the functionality of the child chain and all interfaces that would allow the interaction between parent chain and child chain.

## Scope

- setup requirements
- interface of contracts
- receipt structure 

## Deliverables

- documentation as markdown in `docs` folder of repo

## Bounty Owner/Gardener

@johannbarbie as ChildChainDevLink

## Roles
bounty gardener: @johannbarbie  / 10%
bounty worker: @johannbarbie  / 70%
bounty reviewer: @eshavkun  / 20%

## Gardener checklist

- [ ] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves #4

### CODE REVIEW

File: docs/childchain.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
...
+
+the amount is rounded down to 8 decimals first. rounded amount: `12341234120000000000`
+
+The payload yields the hash: `0x53bdb2747fbb47101844b4d1f5153fda18ecb4c1877504213763d4aa06f5d3b9`
+
+The byte-array encoding of the payload passed to keccak256 is:
+`0x000000000000000000000000000000000000000000000000000000000000000033441122334411223344112233441122334411223344112233441122334411220000000000000000000000006677889900112233445566778899001122334455000000000000000000000000000000000000000000000000AB44DF0BA487D000`
+
+
+## Setup Steps
```
Comment (eshavkun) [inline docs/childchain.md:66]: Those steps look more like a template than actual guide. Is it supposed to be like that?

**Review by eshavkun — COMMENTED** [review]
The bounty mentions `interface of contracts` in the Scope but cannot find it here

File: docs/childchain.md (hunk 1 above)
Comment (johannbarbie) [reply docs/childchain.md:66]: yeah, we don't know more yet, will need to work out this bounty first: https://github.com/leapdao/nervos/issues/11

**Review by eshavkun — APPROVED** [review]
Looks good

**Pull request merged**